#!/usr/bin/env python
PACKAGE = "robot_arm_motion"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("y_offset", double_t, 0, " offset on the x-direction", -0.3, -2, 2)
gen.add("x_offset", double_t, 0, " offset on the y-direction", 0.0, -2, 2)
gen.add("z_offset", double_t, 0, " offset on the z-direction", 0.3, -2, 2)

exit(gen.generate(PACKAGE, "markerTarget", "targetOffsets"))